### PR TITLE
Fixes the Font-size which are hard-coded and make it responsive for small screen

### PR DESCRIPTION
**Pull request by arpit456jain**
## Related Issue or bug

The font size is hard-coded in every tag like h1 and p , it is ok for desktop version but in small screen like phone the text is not clearly visible It is looking like this.
Fixes: #186 

#### Describe the changes you've made
I had replace all the Inline Css to Internal Css as much as possible and Added media queries for Small Screen
Now the website is looking like this

https://user-images.githubusercontent.com/55352601/112106950-56c4d580-8bd4-11eb-826f-773d92542f41.mp4

Please Review it and merge it before any other PR got merged otherwise I have to do all changes again which will be very difficult for me.

## Type of change

Please delete options that are not relevant.

- [x] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [x] Code style update (formatting, local variables)
- [x] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] This change requires a documentation update

## How Has This Been Tested?
I had uploaded a video above.
Please describe the tests that you ran to verify your changes. Provide instructions so we can reproduce. 

#### Describe if there is any unusual behavior of your code(Write `NA` if there isn't)
A clear and concise description of it.

## Checklist:

- [x] My code follows the style guidelines of this project.
- [x] I have performed a self-review of my own code.
- [x] I have commented my code, particularly in hard-to-understand areas.
- [x] I have made corresponding changes to the documentation.
- [x] My changes generate no new warnings.
- [x] I have added tests that prove my fix is effective or that my feature works.
- [ ] New and existing unit tests pass locally with my changes.
- [ ] Any dependent changes have been merged and published in downstream modules.

## Screenshots

 Original           | Updated
 :--------------------: |:--------------------:
 **original screenshot**  | <b>updated screenshot </b> |

### CODE REVIEW

**Comment by arpit456jain** [comment]
@swapnilsparsh sir Please review it and Please add the labels of gsscoc21 before merging. Sir I had replaced a lot of inline-CSS **(hard-coded)** with internal CSS It will be very difficult for me do this changes again It took  more than 4 hours for me Please merge it as soon as possible before any conflicts occur

**Comment by arpit456jain** [comment]
@swapnilsparsh sir please add GSSoC'21 label before merging

**Comment by arpit456jain** [comment]
@swapnilsparsh sir please review it.

**Comment by arpit456jain** [comment]
@swapnilsparsh  sir please add the label of gsscoc'21 before merging it

**Comment by sakshikhachane** [comment]
Hey @arpit456jain,
Please have patience. We all are students and it requires time to review. If we didn't review the PR in 2days then you can ask us on the discord channel.

**Comment by arpit456jain** [comment]
Sorry @sakshikhachane mam 
Actually last time when I created a PR on a different project  it was ok with no conflicts but next day any other PR got merge and some conflicts occur in PR that's why i have to do changes again.
But in this project i had done a lots of correction which will be very difficult for me to do again.

File: main_app/templates/main_app/common.html [hunk 1]
```diff
@@ -138,7 +124,45 @@
         border-bottom: 0;
         border-left: 0.3em solid transparent;
       }
-
+      /*css added by arpit after removing inline CSS */
```
Comment (sakshikhachane) [inline main_app/templates/main_app/common.html:127]: Remove this line.

File: main_app/templates/main_app/common.html [hunk 2]
```diff
@@ -138,7 +124,45 @@
...
+    nav ul a.btn:hover
+    {
+      background-color :#dc3545;
+      color:#fff;
+    }
+    .footer 
+    {
+      background-color: rgb(21, 48, 78);
+    }
+    /*arpit css end*/
```
Comment (sakshikhachane) [inline main_app/templates/main_app/common.html:165]: Remove this comment.

File: main_app/templates/main_app/home.html [hunk 3]
```diff
@@ -143,6 +143,68 @@
         transform: rotate(360deg);
```
Comment (sakshikhachane) [inline main_app/templates/main_app/home.html:143]: Because of the changes in the `home.html` page, the gif image is going inside the nav. Fix it.
image

File: main_app/templates/main_app/women_laws.html [hunk 4]
```diff
@@ -12,8 +12,6 @@
             background-color: black;
```
Comment (sakshikhachane) [inline main_app/templates/main_app/women_laws.html:12]: The fonts for the nav should be as previous. Undo the changes that are done.
image

File: main_app/templates/main_app/women_rights.html [hunk 5]
```diff
@@ -46,85 +43,128 @@
...
+            padding: 1.4rem;
+            border-radius: 25rem;
+        }
+
+        #myBtn:hover {
+            background-color: white;
+            background-image:url("{% static 'Images/toph.png' %}");
+        }
+
+        /*Css added by arpit*/
```
Comment (sakshikhachane) [inline main_app/templates/main_app/women_rights.html:75]: Remove this comment in every file.

**Review by sakshikhachane — CHANGES_REQUESTED** [review]

File: main_app/templates/main_app/common.html (hunk 2 above)
Comment (arpit456jain) [reply main_app/templates/main_app/common.html:165]: ok mam

**Comment by arpit456jain** [comment]
@sakshikhachane mam all the requested changes are done. before I commit this changes
I have a request.
 Please review navbar font-size in this video. I think somehow I had missed to add changes on that particular a tag when anyone is logged in....
sorry for that.

https://user-images.githubusercontent.com/55352601/112326522-9b3a9900-8cda-11eb-9618-63413b5a2ec2.mp4


I hop now font-sizes are perfect.

**Comment by arpit456jain** [comment]
I had increase the font-size from 2rem to 2.5 rem and added some boldness now take a look .

https://user-images.githubusercontent.com/55352601/112329182-fe2d2f80-8cdc-11eb-9435-d69f2f4b5248.mp4

**Comment by arpit456jain** [comment]
@sakshikhachane mam may I commit the changes.??

**Comment by sakshikhachane** [comment]
> @sakshikhachane mam may I commit the changes.??

Sure!

**Comment by arpit456jain** [comment]
ok @sakshikhachane  mam but how to solve this conflict which is occurring now ?

**Comment by arpit456jain** [comment]
Thank God The conflict resolved .
I am getting stressed for a sec I thought I have to do the changes again.
@sakshikhachane mam please review it.

**Review by swapnilsparsh — CHANGES_REQUESTED** [review]
1. Fix the navbar black colour.
black
black2

2. Fix the login page
login

3. Ngo Page missing
ngo

**Comment by swapnilsparsh** [comment]
@arpit456jain 
Don't spam and please have patience.
There are lot of bugs in your PR.
Firstly rebase your code and then resolve the bugs.

**Comment by arpit456jain** [comment]
@swapnilsparsh sir all the requested changes are been done but I am not able to find where the color of navbar is not black .Can you please mention the page.

I have pull the latest code merge it in my branch.
I am attaching a video Please review it.

https://user-images.githubusercontent.com/55352601/112601116-0f3e8380-8e38-11eb-9bc9-cb085c719b8d.mp4

**Comment by arpit456jain** [comment]
@sakshikhachane mam can you please review this if further change has to be done.

**Comment by arpit456jain** [comment]
@swapnilsparsh sir please review it

**Comment by swapnilsparsh** [comment]
> @swapnilsparsh sir please review it

https://github.com/swapnilsparsh/Rescue/pull/195#pullrequestreview-621395974

**Comment by arpit456jain** [comment]
@swapnilsparsh sir when i commit changes you requested there were no conflicts 
May be any PR got merged .

**Comment by swapnilsparsh** [comment]
> @swapnilsparsh sir when i commit changes you requested there were no conflicts
> May be any PR got merged .

You have to fix this https://github.com/swapnilsparsh/Rescue/pull/195#pullrequestreview-621395974

**Comment by arpit456jain** [comment]
@swapnilsparsh sir Its 2:33 am now but i am still working just because I had done a lot of work on this issue.
Finally I had again pull latest code
commit it
and done some required changes.
now it has no conflicts
so I am requesting you @swapnilsparsh sir please review it and merge it as soon as possible.

**Comment by arpit456jain** [comment]
This is how the website is looking now after my above commits

https://user-images.githubusercontent.com/55352601/112735106-1cf72480-8f70-11eb-8d37-c82fabafa073.mp4

@swapnilsparsh sir please add labels of level3 if possible.

**Comment by arpit456jain** [comment]
@swapnilsparsh please review it.

**Review by swapnilsparsh — CHANGES_REQUESTED** [review]
navbar

Everything is fine but you have to fix the navar colour as it was before.
In picture you can see the black boundary, fix it.

**Comment by arpit456jain** [comment]
@swapnilsparsh I will fix it within 5 min .

**Comment by arpit456jain** [comment]
res

**Comment by arpit456jain** [comment]
@swapnilsparsh done.

**Review by swapnilsparsh — CHANGES_REQUESTED** [review]
not fix

For small screen not fix yet.

**Comment by arpit456jain** [comment]
Untitled

**Comment by arpit456jain** [comment]
@swapnilsparsh done 
and this time I had  re-check all pages in desktop and mobile version both.

**Comment by arpit456jain** [comment]
@swapnilsparsh please add label of level   before merging.

**Comment by arpit456jain** [comment]
@swapnilsparsh kindly review it.

**Review by swapnilsparsh — CHANGES_REQUESTED** [review]
1. Fix the font for NGO page also.
2. While checking for small screen, login page footer has bug, fix it.

**Comment by arpit456jain** [comment]
@swapnilsparsh I am sorry I am not able not fix these issues I am closing this PR kindly assign issue #186  to someone else.
